back-nest/Dockerfile

```diff
@@ -30,6 +30,12 @@ COPY . .
 
 EXPOSE 3000
 
+RUN npm run prisgen
+
 RUN npm run build
 
+ENTRYPOINT ["sh", "-c", "npm run migrate && npm run start"]
+
 CMD ["npm", "run", "start:dev"]
```

Comment:
Why do you keep Enrtypoint and CMD with startup command? Also, don't use start:dev in docker. Intead use `npm run start`